float64 roll
float64 pitch
float64 yaw
float64 roll_rate
float64 pitch_rate
float64 yaw_rate

float64 roll_des
float64 pitch_des
float64 yaw_des
float64 roll_rate_des
float64 pitch_rate_des
float64 yaw_rate_des

float64 roll_acc
float64 pitch_acc
float64 yaw_acc

float64 roll_real
float64 pitch_real
float64 yaw_real